docs/tutorials/building-your-first-bot.mdx

```diff
@@ -0,0 +1,342 @@
+This will allow our player to be controlled by the bot logic when it is set to true. However, there is an existing `Player`
+GameObject in the scene, which we want to control using our manual input controls. Click this `Player` in the object hierarchy,
+find the `Player Input Control` component, set `Is Bot` to unchecked, and then save (you may find that it is already unchecked).
+
+![Screenshot of the player prefab with the components added.](./img/building-your-first-bot/is_bot.png)
+
+That's it for actions! A bot may now call into two actions (moving and attacking), which we will demonstrate in the section
+when we build our bot.
+
+## Using a ready-to-go bot from the Bot Marketplace
+
+Now that our integration is complete, our final step is to use an implementation of a bot. We will start with a template
+found in our [Bot Marketplace](https://play.regression.gg/marketplace), and then will show you where to edit its code.
+
+Go to [this page](https://play.regression.gg/marketplace/1) to see the existing bot, and click "Create Bot" to create a copy 
+of the bot. On the bot creation page, give it a name such as "KitingBot", and then click "Create Bot". You will then be
+taken to the Agent Builder tool; feel free to click around and see how the bot works!
+
+![Screenshot of the bot creation page.](./img/building-your-first-bot/marketplace_bot.png)
+
+![Screenshot of the bot creation page.](./img/building-your-first-bot/create_bot_9.png)
+
+![Screenshot of the agent builder configuration for this bot.](./img/building-your-first-bot/agent_builder.png)
+
+Go back to Unity, and within the **Regression Games** menu in the menu bar, click **Synchronize Bots with RG**. This will pull your new
+bot into your Unity project. 
+
+![Screenshot of the agent builder configuration for this bot.](./img/building-your-first-bot/synchronize_bots.png)
+
+At this point, you can click play, and then select the Regression Games overlay button in the bottom
+right of your game screen. Use the dropdown to select the bot you just created, and then click "Start Bot". The bot will begin to
+run around and shoot the enemy!
+
+![Screenshot of the bot creation page.](./img/building-your-first-bot/start_bot_10.png)
+
+You now have a working bot, fully integrated into your game! If you want to modify the behavior of the bot, you can make edits
+within the Agent Builder, or edit the code in your Unity project at `Assets/RegressionGames/Runtime/Bots/BOTNAME/Nodes`
+
+## Next Steps
+
+Congratulations, you've now experienced the integration process for bots! Using this guide and our reference material,
+you can now integrate bots into your own game. If you have any questions, please reach out to us on our 
+[Discord](https://discord.com/invite/925SYVse2H). Please note that we offer **dedicated support** for your integration, 
+we are happy to do it for you or walk you through it on a call step by step.
```

Comment:
```suggestion
We are happy to work directly with your project for you, or walk you step-by-step through the integration process over a call.
```

Reply:
Maybe "work directly on your project for you?" I was thinking with your project sounds weird since we aren't working with the project, we are working on it. (as I type this, grammarly is trying to correct "with" to "on" as well)